Java/java-selenium/src/test/java/example/exampleTest.java

```diff
@@ -0,0 +1,70 @@
+package example;
+
+import java.io.File;
+import java.io.IOException;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import io.github.bonigarcia.wdm.WebDriverManager;
+import com.deque.html.axecore.results.Results;
+import com.deque.html.axedevtools.selenium.*;
+import com.deque.html.axedevtools.selenium.reporter.*;
+
+
+
+public class exampleTest {
+    static String reporter = new File("src/test/resources/reporter-cli-macos").getAbsolutePath();
+    public static AxeReportingOptions _reportOptions = new AxeReportingOptions();
+    public static WebDriver driver = null;
+    public static AxeDriver axedriver = null;
+    public static AxeSelenium axeSelenium = null;
+    public static String Logger;
+    public static String Destination;
+
+    @BeforeClass
+    public static void initiate_drivers(){
+        _reportOptions = new AxeReportingOptions();
+        WebDriverManager.chromedriver().setup();
+        driver = new ChromeDriver();
+        axedriver = new AxeDriver(driver);
+        axeSelenium = new AxeSelenium();
+        AxeConfiguration.configure()
+                .testSuiteName("Test")
+                .outputDirectory("axe-json-reports/");
+        Logger =  new File("axe-json-reports/").getAbsolutePath();
+        Destination =  new File("a11y-results/").getAbsolutePath();
+
+    }
+
+    @Test
+    public void test_a11y_workshop_homepage() {
+        driver.get("https://broken-workshop.dequelabs.com/");
+        Results results1 = axeSelenium.logResults(_reportOptions.uiState("Homepage_scan")).run(axedriver);
+    }
```

Comment:
No we're not asserting, because we know that there are a11y issues on the page that we're testing. We have 2 tests in each of the example projects, 1 on the homepage and the other on the recipe popup. If we assert, the test would fail, the build would fail and the 2nd test would not be run.